doctr/utils/common_types.py

```diff
@@ -3,10 +3,10 @@
 # This program is licensed under the Apache License version 2.
 # See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
 
-from typing import Tuple
+from typing import Tuple, List
 
 __all__ = ['Point2D', 'BoundingBox', 'Polygon4P']
 
-Point2D = Tuple[float, float]
-BoundingBox = Tuple[Point2D, Point2D]
-Polygon4P = Tuple[Point2D, Point2D, Point2D, Point2D]
+Point2D = List[float]
+BoundingBox = Tuple[float, float, float, float, float]
+Polygon4P = List[Point2D]
```

Comment:
If we still have polygons with 4 points, I suggest keeping the old one and create a bare `Polygon = List[Point2D]`